### PR TITLE
[bitnami/redmine] Inject Root CA into image

### PR DESCRIPTION
**Pull request by ballj**
**Description of the change**

Add a certificate authority into the redmine image. Pulls CA from kubernetes secret using a sidecar init container.

**Benefits**

- Allows LDAP authentication using LDAPS to servers with an internal certificate.
- Adds a snakeoil certificate to the container. Used can be used for TLS sessions to the ingress.

**Possible drawbacks**

None

**Applicable issues**

None

**Additional information**

None

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/redmine/values.yaml [hunk 1]
```diff
@@ -470,3 +470,17 @@ mailReceiver:
...
+# Add custom certificate authorities to redmine container
+certificates:
+  customCA: []
+  # - secret: custom-CA
+  # - secret: more-custom-CAs
+  image:
+    repository: alpine
+    # pullPolicy:
+    # pullSecrets
+    #   - myRegistryKeySecretName
```
Comment (dani8art) [inline bitnami/redmine/values.yaml:483]: Could you modify this following the standard binami image structure in `values.yaml`? Please take a look at https://github.com/bitnami/charts/tree/master/bitnami/common#imageroot

Also, please use `minideb` as https://github.com/bitnami/charts/blob/master/bitnami/ghost/values.yaml#L42

**Comment by dani8art** [comment]
Hi @ballj , thank you so much for the contribution, it is a good feature! 

I have one question, would this cover the case where the user wants to provide the cert.pem and the key.pem? I think we should cover this as well.

**Comment by ballj** [comment]
Hi @dani8art 
Will modify the structure as suggested

Can you clarify what you mean with the cert.pem and key.pem? Is this for replacing the snakeoil cert?

**Comment by javsalgar** [comment]
Hi,

I don't have much experience with Redmine, but I suppose that this CA is used to create certificates (with public key and private key), am I correct? What if the user wants to provide these as well?

File: bitnami/redmine/values.yaml (hunk 1 above)
Comment (ballj) [reply bitnami/redmine/values.yaml]: These have been added along with the ability to use custom certificates

**Comment by dani8art** [comment]
Waiting for CI/CD

**Pull request merged**